apps/buildog/src/app/(auth)/layout.tsx

```diff
@@ -1,13 +1,12 @@
-import { Metadata } from "next";
+"use client";
 import Link from "next/link";
-import { SignUpForm } from "@/components/user-sign-up-form";
+import { ReactNode } from "react";
 
-export const metadata: Metadata = {
-  title: "Sign Up",
-  description: "Sign up for a new account on Buildog",
-};
+interface AuthLayoutProps {
+  children: ReactNode;
+}
```

Comment:
We have only need a one prop here. We don't need to generate an interface for this we can directly give the type in function.